app/helpers/organisations_helper.rb

```diff
@@ -2,14 +2,38 @@
 
 module OrganisationsHelper
   def organisation_logo(organisation)
-    return unless organisation && organisation.name.present?
+    return unless organisation && (organisation.name.present? || organisation.slug.present?)
 
     if organisation.name == 'Department for Work and Pensions'
-      render partial: 'shared/dwp_logo', locals: { organisation: }
-    elsif File.exist?("app/assets/images/#{organisation.name.parameterize(separator: '')}.png")
-      image_tag("#{organisation.name.parameterize(separator: '')}.png", class: 'govuk-image', alt: organisation.name)
+      render_dwp_logo(organisation)
     else
-      content_tag(:span, organisation.name)
+      render_organisation_logo(organisation)
     end
   end
+
+  private
+
+  def render_dwp_logo(organisation)
```

Comment:
This all feels more complicated to read than it was before, are we actually changing much in the implementation?

Reply:
In fact, as we are using titleize, we can probably go back to the original implementation as the errors only started after the slug presence checks came in

Reply:
Ahh rubocop 😄 okay let's see if it works as before with titleize. Otherwise if you can make it easier to read then great, or just leave it as it is 👍